wunderio/client-UK-FSA-beta
# Feature/fsa 928 inline field prefix

To test:

- Go to subscribe and profile pages
- Upon reaching it, ensure 'SMS' checkbox is checked
- '+44' prefix should be inline with its input
